hack/update-metadata.sh

```diff
@@ -0,0 +1,75 @@
+#!/bin/bash
+
+set -o errexit
+set -o nounset
+set -o pipefail
+
+# Usage:
```

Comment:
This was a bit unclear to me at first. The script can run in two modes, either no argument is provided and in that case it takes ocp version from `.channels[0].currentCSV` found in `local-storage-operator.package.yaml` and updates just the metadata accordingly, or it takes an argument with desired ocp version and updates both metadata **and** the `local-storage-operator.package.yaml` file.

This makes the semantics of the argument slightly unclear but I don't think it's worth investing more time in utility scripts like this one, however some nicer description would help our future selves.